packages/office-ui-fabric-react/src/components/OverflowSet/OverflowSet.base.tsx

```diff
@@ -32,10 +27,12 @@ export class OverflowSetBase extends BaseComponent<IOverflowSetProps, {}> implem
   }
 
   public render(): JSX.Element {
-    const { items, overflowItems, className, focusZoneProps, styles, vertical, role, doNotContainWithinFocusZone } = this.props;
+    const { items, overflowItems, className, focusZoneProps, styles, vertical, doNotContainWithinFocusZone } = this.props;
 
     this._classNames = getClassNames(styles, { className, vertical });
 
+    const role = this.props.role || vertical ? 'menu' : 'menubar';
```

Comment:
Isn't it weird that you apply `role='menu'` when people specify their own role?

Reply:
if this.props.role, then that role is used, otherwise we use menu or menubar depending on vertical prop.